sass/_theme_rst.sass

```diff
@@ -298,9 +298,25 @@
       @extend .fa-download
       &:before
         margin-right: 4px
+  .wy-table-responsive
+    table
+      td
+        white-space: normal
+
+
+// Tablet specific
++media($tablet)
+  .wy-table-responsive
+    table
+      td
+        white-space: nowrap !important
```

Comment:
We shouldn't need this block twice, and the `!important` flag should be avoided. Ordering this correctly will solve this issue.